ui/components/app/Summary/index.tsx

```diff
@@ -1,21 +1,38 @@
-import useSWR from 'swr';
-import type { AnalysesResponse } from '@/pages/api/analyses';
-import { Roarr } from 'roarr';
+import React from 'react';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import LightbulbIcon from '@mui/icons-material/Lightbulb';
+import WarningIcon from '@mui/icons-material/Warning';
+import ErrorIcon from '@mui/icons-material/Error';
+import CrisisAlertIcon from '@mui/icons-material/CrisisAlert';
+import DarkModeIcon from '@mui/icons-material/DarkMode';
+import { AnalysesSummary, Severity } from 'lib/findings';
 
-export const Summary = () => {
-  const { isLoading, error } = useSWR<AnalysesResponse>('/api/analyses', {
-    onSuccess(data) {
-      Roarr.info(JSON.stringify(data));
-    },
-  });
+type SummaryProps = AnalysesSummary;
+const CriticalIcon = CrisisAlertIcon;
+const HighIcon = ErrorIcon;
+const MediumIcon = WarningIcon;
+const LowIcon = DarkModeIcon;
+const NoneIcon = LightbulbIcon;
 
-  if (isLoading) {
-    return <div>Loading...</div>;
-  }
+export const Summary = ({ faults, findings, scansPassed, severityCounts }: SummaryProps) => {
+  const summaryTitle = faults
+    ? <>{faults} Faults ({findings} Findings)</>
+    : <>No problems detected</>;
```

Comment:
Could just be a template string instead of embedded Fragments, which is (marginally) less code and less function calls client-side:
```suggestion
  const summaryTitle = faults
    ?`${faults} Faults (${findings} Findings)`
    : 'No problems detected';
```